torch/_torch_docs.py

```diff
@@ -8174,6 +8174,54 @@ def merge_dicts(*dicts):
 """,
 )
 
+add_docstr(
+    torch.narrow_copy,
+    r"""
+narrow_copy(input, dim, start, length, *, out=None) -> Tensor
+
+Same as :meth:`Tensor.narrow` except returning a copy rather
+than shared storage.  This is primarily for sparse tensors, which
+do not have a shared-storage narrow method.  Calling ``narrow_copy``
+with ``dimemsion > self.sparse_dim()`` will return a copy with the
+relevant dimension narrowed, and ``self.shape`` updated accordingly.
+
+Args:
+    input (Tensor): the tensor to narrow
+    dim (int): the dimension along which to narrow
+    start (int): the starting dimension
```

Comment:
This should probably be "starting offset" as the `dim` argument is the only dimension "starting dimension" is not correct.

Reply:
This was copied from `narrow` I'll update that too.